servers/plugins/org.fusesource.ide.server.karaf.ui/plugin.xml

```diff
@@ -44,6 +54,16 @@
             id="org.fusesource.ide.server.karaf.ui.server.fragment.30"
             typeIds="org.fusesource.ide.karaf.server.30">
       </fragment>
+      <fragment
+            class="org.fusesource.ide.server.karaf.ui.runtime.v2x.KarafServerWizardFragment2x"
```

Comment:
strange to have the 2x in the name although it is for 3 and 4 version too

Reply:
I agree. Maybe we should consider a refactoring in future. (out of scope of this PR)

Reply:
have you created a JIRA task for it?

Reply:
https://issues.jboss.org/browse/FUSETOOLS-2528